docs/extra-build-config.md

```diff
@@ -261,6 +261,18 @@ In order to use Pican2 CAN module, set the following variables:
 
 See: <http://skpang.co.uk/catalog/pican2-canbus-board-for-raspberry-pi-23-p-1475.html>
 
+## Enable infrared
+
+Users who want to enable infrared support, for example for using LIRC (Linux
+Infrared Remote Control), have to explicitly set in local.conf:
+
+    ENABLE_IR = "1"
+
+This will add device tree overlays gpio-ir and gpio-ir-tx will to config.txt.
```

Comment:
Small typo: "will".

Reply:
Thank you. Fixed. I've forced pushed a new version for review. Thanks, Leon